src/helpers/helpers.h

```diff
@@ -47,7 +47,7 @@
             input /= base;
             (c < 10) ? c += '0' : c += 'A' - 10;
             result = c + result;
-        } while (input);
+        } while (input != 0);
```

Comment:
Given that this code is dealing with characters, I would rather it say while (input != '\0');.

That being said, does this function even need to exist?  Would the above template cover the uint64_t conversion to string?  I think this is worth testing.